src/Uno.UI/UI/Xaml/UIElement.cs

```diff
@@ -343,7 +343,7 @@ internal static Matrix3x2 GetTransform(UIElement from, UIElement to)
 					offsetY = layoutSlot.Y;
 				}
 
-#if !UNO_HAS_MANAGED_SCROLL_PRESENTER
+#if !UNO_HAS_MANAGED_SCROLL_PRESENTER && !__WASM__
```

Comment:
Looks like there's a missing #define for this special case...

Reply:
Right ... but finally I just removed it!